manifests/h/Hashicorp/Vagrant/2.3.3/Hashicorp.Vagrant.installer.yaml

```
@@ -0,0 +1,38 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Hashicorp.Vagrant
+PackageVersion: 2.3.3
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Custom: /norestart
+InstallerSuccessCodes:
+- 3010
+- 1706
+UpgradeBehavior: install
+InstallLocationRequired: true
+RequireExplicitUpgrade: true
+DisplayInstallWarnings: false
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64

```

```suggestion
- Architecture: x64
```
Is the installer specific to only `en-US` locale? Previous manifest's metadata show that maybe this is not required.